v63002/inref/gtm8165.m

```diff
@@ -0,0 +1,20 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;								;
+; Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	;
+; All rights reserved.						;
+;								;
+;	This source code contains the intellectual property	;
+;	of its copyright holder(s), and is made available	;
+;	under a license.  If you do not know the terms of	;
+;	the license, please stop and do not read further.	;
+;								;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;
+tpnotacid
+	tstart ():(serial:transaction="BA")
+	if $trestart>2 write /wait(.999)
```

Comment:
Also need a test of the no-timeout case. Should behave as if the maximum possible timeout has been specified. And needs mention in the reference file.

Reply:
The no-timeout case can be folded into the existing timeout case by randomly choosing a timeout that is greater than .123 or no timeout. Both should have the same effect.